docs/manage/enabling_surveys_using_config_files.md

```diff
@@ -0,0 +1,537 @@
+
+5. Now that the survey definitions are done, we can assign them to the buttons where we want them to be used. OpenPATH currently supports the following buttons:
+
+| Button Name | `key-name` | Location on app | Special notes |
+:---:|:---:|:---:|:---:
+| Trip Button | `"trip-button": {...}` | This button shows up at the bottom of the `<infinite-scroll-trip-item>`| This button will not show up in the user-interface UNLESS you define `"trip-button": {...}` as a key within the `"buttons": {...}` key object |
+| Place Button | `"place-button": {...}` | This button shows up between 2 trips as a `<infinite-scroll-place-item>` | The entire `<infinite-scroll-place-item>` element will not show up in the user-interface UNLESS you define `"place-button": {...}` as a key within the `"buttons": {...}` key object |
+
+6. For each button you wish to include in your Study/Program, add a key object as a child of `"buttons"`. The name you give your button object MUST match that of the `key-name` in the table above.
+
+```json
+"buttons": {
+    "key-name1": {
+        //HERE
+    },
+    "key-name2": {
+        //HERE
+    }
+}
+```
+
+7. The keys contained inside of each survey object are **important**. Here is the template to follow, and we will explain what each field is for:
+
+```json
+"key-name1": {
+    "surveyName": "",
+    "not-filled-in-label": {
+        "en": "",
+        "es": ""
+    },
+    "filled-in-label": {
+        "en": "",
+        "es": ""
+    }
+}
+```
+
+- **`"surveyName":`** defines the survey that will display when this button is clicked
+- **`"not-filled-in-label":`** contains the internationalizations for the text displayed on the button when the survey **has not** been filled out yet.
+  -  **`"en":` / `"es":`/ etc.** contains the text which will be displayed on the button when the survey **has not** been filled out yet. For example, this might read `"Take my survey"`
+- **`filled-in-label":`** contains the internationalizations for the text displayed on the button when the survey **has** been filled out.
+  -  **`"en":` / `"es":`/ etc.** contains the text which will be displayed on the button when the survey **has** been filled out. For example, this might read `"Response submitted"`
+
+8. Finally, you will define how the trip-labels (e.g. the labels for mode and purpose of each trip) will be collected. There are only 2 options currently supported by OpenPATH: `MULTILABEL` and `ENKETO`.
+
+```json
+...
+"trip-labels": "MULTILABEL"
+...
+```
+or
+```json
+...
+"trip-labels": "ENKETO"
+...
+```
+
+9. Now that your config file has been completed, you can compare it to some of our examples to make sure you have written it correctly:
+
+# Examples of configs with varying surveys _(for reference)_
+
+## Basic Config Template _(without any buttons or surveys, and MULTILABEL for trip-labels)_
+----
+`[your program name].nrel-op.json`
```

Comment:
I think you should link to the configs here instead of copy-pasting them.
If you copy-paste, you will need to re-copy-paste every time we change the format.
It is also redundant and does not follow the DRY principle